smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ClientRuntimeTypes.kt

```diff
@@ -66,6 +66,8 @@ object ClientRuntimeTypes {
         val LoggerMiddleware = runtimeSymbol("LoggerMiddleware")
         val ContentLengthMiddleware = runtimeSymbol("ContentLengthMiddleware")
         val ContentMD5Middleware = runtimeSymbol("ContentMD5Middleware")
+        val FlexibleChecksumsRequestMiddleware = runtimeSymbol("FlexibleChecksumsRequestMiddleware")
+        val FlexibleChecksumsResponseMiddleware = runtimeSymbol("FlexibleChecksumsResponseMiddleware")
```

Comment:
I don't see where these middlewares are ever added to the stack

Reply:
Middleware are added in aws-sdk-swift PR